app/test_main.py

```diff
@@ -1,3 +1,92 @@
+from typing import Any, Type
+
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatDogYears:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_age",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="test should return zero ages when cat and dog age == 0"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="test should return zero ages when cat and dog age < 15"),
+            pytest.param(
+                15,
+                15,
+                [1, 1],
+                id="test should return 1 ages when cat and dog age >= 15"
+            ),
+            pytest.param(
+                23,
+                23,
+                [1, 1],
+                id="test should return 1 ages when cat and dog age <= 23"
+            ),
+            pytest.param(
+                24,
+                24,
+                [2, 2],
+                id="test should return 2 ages when cat and dog age == 24"
+            ),
+            pytest.param(
+                27,
+                27,
+                [2, 2],
+                id="test should return 2 ages when cat and dog age == 27"
+            ),
+            pytest.param(
+                28,
+                28,
+                [3, 2],
+                id=""
+                   "test should return "
+                   "3 cat age and 2 dog age "
+                   "when cat and dog age == 28"
+            ),
+            pytest.param(
+                -1,
+                15,
+                [0, 1],
+                id="test should return zero when some age < 0"
+            )
```

Comment:
This test case is not clear. If the function is supposed to handle negative numbers, it should be clearly stated in the function's requirements. If negative numbers are not valid input, the function should raise an error, and this should be tested in the error handling tests.